hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -305,7 +304,8 @@ interface IOmdbInsightsState {
   nextClickable: boolean;
   includeFso: boolean;
   includeNonFso: boolean;
-  prevClickable: boolean
+  prevClickable: boolean;
+  pageDisplayCount: number;
```

Comment:
Could you help me understand what this variable is for? Maybe we could add a comment here, it is used later at many places and for me it's not straightforward what does it stand for.

Reply:
Added comment in latest commit we are navigating need to understand we are at last  previous page and pageDisplayCount is used to store values For ex Page 1, Value =10, Page 2= Value 20

Reply:
This is for one tab? And if we are navigating between tabs (eg. from open keys to keys pending for deletion) it will be cleared and used for that?

Reply:
They are unique across all tabs in changeTab function we are setting to default 1 pageDisplayCount: 1.